src/components/package-search-list.js

```diff
@@ -28,6 +28,7 @@ export default function PackageSearchList({
             pathname: `/eholdings/packages/${item.content.id}`,
             search: location.search
           }}
+          active={(router.route.match.params.type === 'packages') && (router.route.match.params.id === (item && item.content && item.content.id))}
```

Comment:
This is a little long, could we pull this (or part of it) into a variable above the `return`?

Something like
```js
let { params: routeParams } = router.route.match;
let isPackagePage = routeParams.type === 'packages';

...

active={(isPackagePage && routeParams.id == (item.content && item.content.id)}
```

Also, `item` will always be provided, so we just need to verify `item.content` exists, no need to verify `item`.